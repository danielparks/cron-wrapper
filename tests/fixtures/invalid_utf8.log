Command: ["tests/fixtures/invalid_utf8.sh"]
Start: 2023-07-05T11:07:40.911008000-07:00

0.003 out bad â(¡ bad
0.003 exit 0
