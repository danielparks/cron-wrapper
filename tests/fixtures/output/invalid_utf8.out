Command: ["tests/fixtures/invalid_utf8.sh"]
Start: 2023-07-26T13:32:22.288640000-07:00

0.003 out bad â(¡ bad
0.003 exit 0
